packages/adapter-vercel/image-loader.d.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * https://vercel.com/docs/concepts/image-optimization
+ */
+export default function loader(
+	src: string,
+	width: number,
+	loader_options: any,
```

Comment:
I think this needs to be removed to match the actual code. Can we update the code to use this type? I think it would have avoided this issue
```suggestion
```